docs/dymos_book/examples/multi_phase_cannonball/multi_phase_cannonball.ipynb

```diff
@@ -379,7 +379,8 @@
     "# In this case, we connect 'm' to pre-existing input parameters\n",
     "# named 'mass' in each phase.\n",
     "traj.add_parameter('m', units='kg', val=1.0,\n",
-    "                   targets={'ascent': 'mass', 'descent': 'mass'}, static_target=True)\n",
```

Comment:
How did these tests pass before? Is the default value of mass the same as the value set in the problem?

Reply:
Honestly not sure, that must be the case.